.wp-env.json

```diff
@@ -1,4 +1,7 @@
 {
     "core": null,
-    "plugins": ["."]
+    "plugins": ["."],
+    "config":{
+        "SUCURISCAN_API_URL": "https://sucuri.net"
```

Comment:
Should `SUCURISCAN_API_URL` be set to `https://sucuri.net`?

Reply:
This is just a placeholder url for the e2e tests